uaclient/cli.py

```diff
@@ -1772,6 +1779,17 @@ def action_help(args, *, cfg):
     return 0
 
 
+def _warn_about_new_version(cmd_args=None) -> None:
+    # If no args, then it was called from the main error handler.
+    # We don't want to show this text for the "api" CLI output.
```

Comment:
We also don't want to show this for commands with `--format=json` right?

Reply:
good point - will hide it

Reply:
done